src/constants.ts

```diff
@@ -51,3 +51,217 @@ export const AMM_TYPES = {
   "5ocnV1qiCgaQR8Jb8xWnVbApfaygJ8tNoZfgPwsgx9kx": "Sanctum S",
   Gswppe6ERWKpUTXvRPfXdzHhiCyJvLadVvXGfdpBqcE1: "GuacSwap",
 };
+
+export const STACK_HEIGHT = {
+  SWAP: 2,
+  TOKEN_TRANSFER: 3,
+};
+
+export const SWAP_IN_OUT_ACCOUNTS_POSITION = {
+  AMM55ShdkoGRB5jVYPjWziwk8m5MpwyDgsMWHaMSQWH6: {
```

Comment:
@jokrsec would there be a test that ascertains every AMM type has a corresponding entry in `SWAP_IN_OUT_ACCOUNTS_POSITION`

Reply:
Sure. Currently updated the mapping to have instruction names as keys. Because in few AMMs multiple instructions(with different source and destination positions) are being considered for swap. I will add test to assert every instruction has corresponding entry